gemet/thesaurus/tests/test_theme_view.py

```diff
@@ -112,3 +112,75 @@ def test_404_error(self):
 
         self.assertEqual(404, resp.status_int)
         self.assertEqual(ERROR_404, resp.pyquery('.error404 h1').text())
+
+
+class TestThemeViewWithUser(GemetTest):
+    def setUp(self):
+        self.theme = ThemeFactory()
+        user = UserFactory()
+        self.user = user.username
+
+    def test_properties(self):
+        PropertyFactory(concept=self.theme, name="prefLabel",
+                        value="prefLabel", status=DELETED_PENDING)
+        PropertyFactory(concept=self.theme, name="prefLabel",
+                        value="prefLabel new", status=PENDING)
+        PropertyFactory(concept=self.theme, name="definition",
+                        value="definition", status=DELETED_PENDING)
+        PropertyFactory(concept=self.theme, name="definition",
+                        value="definition new", status=PENDING)
+        PropertyFactory(concept=self.theme, name="scopeNote",
+                        value="scopenote", status=DELETED_PENDING)
+        PropertyFactory(concept=self.theme, name="scopeNote",
+                        value="scopeNote new", status=PENDING)
+
+        url = reverse('theme', kwargs={'code': self.theme.code,
+                                            'langcode': 'en'})
+        resp = self.app.get(url, user=self.user)
+
+        self.assertEqual(200, resp.status_int)
+        self.assertEqual(resp.pyquery('.content #prefLabel').text(),
+                         "prefLabel new")
+        self.assertEqual(resp.pyquery('.content #definition-text').text(),
+                         "definition new")
+        self.assertEqual(resp.pyquery('.content #scope-note').text(),
+                         "scopeNote new")
+
+    def test_relations_pending(self):
+        self.concept = TermFactory()
```

Comment:
Why is this an instance variable (`self.concept`) and not just a variable (`concept`) ?